vanilla-framework/vanilla-framework
# Update docs/inline images page content

## Done
- Updated copy 👍 
- Updated demo example to showcase live example 😉 
- Check content matches copy doc [here](https://docs.google.com/document/d/1BaY3k01-6zvhC67PMxZD44IKHs4_pTpPm_ix98rHW98/edit#heading=h.momwkyldv4cx)

## QA
- Pull code
- Run /docs > ./run
  - Check http://0.0.0.0:8104/en/patterns/inline-images
- Pull code
- Run ./run
- Review updated examples
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/inline-images/

## Screenshot
![Screenshot 2019-03-28 at 11 49 32](https://user-images.githubusercontent.com/17748020/55155563-9c6a7700-514f-11e9-9018-1c26c50cb2a6.png)
